### DISCUSSION TITLE
Please give me some sample code for do post-processing though useing RenderTarget(off-screen)

### DISCUSSION THREAD

**Discussion by fanbinnet** [google/filament#4069] [Q&A] [answered]
I use ModelViewer load gltf mode, then need do post-processing to change the color of the model.
Can some one can kindly provide me some sample code or some demo.
Thanks !!!

**Comment by romainguy** [comment]
What are you trying to achieve exactly?

**Reply by fanbinnet** [reply]
I plan to load three gltf at the same position, the color of the overlap will be too bright.
I need to lower the color of the over-bright parts and keep the color of the rest of the parts the same.

**Comment by fanbinnet** [comment, marked as answer]
Sample-gltf-bloom is very helpful for my question.But it don't contains in latest code.
I find it in old version code,and it hlep me resolved my problem.

**Reply by prideout** [reply]
Glad it helped.  Note that we replaced that sample with a simpler one (see `samples/rendertarget.cpp`) after we introduced our first-class bloom effect and post-processing framework.
